happy.cabal

```diff
@@ -1,5 +1,5 @@
 name: happy
-version: 1.19.6
+version: 1.19.7
```

Comment:
Let's put this in a separate commit, we might include several changes in a release.